.github/workflows/release.yml

```diff
@@ -6,10 +6,10 @@ jobs:
   publish-egg:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: denolib/setup-deno@master
-      - run: deno install -A -f --unstable -n eggs https://x.nest.land/eggs@0.3.2/eggs.ts
-      - run: |
+     - uses: actions/checkout@v2
+     - uses: denolib/setup-deno@master
+     - run: deno install -A -f --unstable -n eggs https://x.nest.land/eggs@0.3.2/eggs.ts
+     - run: |
```

Comment:
i need to fix this

Reply:
good catch

Reply:
https://github.com/drashland/deno-drash/pull/446/commits/601b33a9b91ae7085bce83a40b310d8224a53187